feature/kl_divergence.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Copyright FMR LLC <opensource@fidelity.com>
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import NoReturn, Tuple
+
+import pandas as pd
+import numpy as np
+
+from scipy.special import kl_div, rel_entr
+from feature.base import _BaseSupervisedSelector, _BaseDispatcher
+from feature.utils import Num, get_task_string, check_true
+
+
+from tqdm import tqdm
+
+
+class _KL_Divergence(_BaseSupervisedSelector, _BaseDispatcher):
```

Comment:
I don't think this needs to uptake Dispatcher. We typically use dispatcher when the method is an umbrella method (like Linear) and the exact implementer is decided eventually upon the parameters (Linear but Lasso etc.)